frontend/src/tests/components/Nav/Footer.test.js

```diff
@@ -14,8 +14,8 @@ describe("Footer tests", () => {
         expect(text.textContent).toEqual('HappierCows is a project of Mattanjah de Vries, Distinguished Professor of Chemistry at UC Santa Barbara. The open source code is available on GitHub.');
     
         const href = screen.getByTestId("github-href");
-        // console.log(href.href);
-        expect(href).toHaveAttribute("href", "https://github.com/ucsb-cs156-f22/f22-5pm-happycows");
+        //console.log(href.href);
```

Comment:
This commented out line was in the original, but we should probably remove it too since it's adjacent to your changes; it shouldn't have been in the code base in the first place.